### DISCUSSION TITLE
[IntelliSense] add option to use postcss.config.js file

### DISCUSSION THREAD

**Discussion by fajaralfa** [tailwindlabs/tailwindcss#14647] [Ideas]
Hi teams, thanks for creating & maintaining tailwind / intellisense.
I want to have an option to use config from postcss.config.js file for intellisense, because i don't really like using multiple config files especially just to make completion work. Or is it already supported? if so, please include the link to the docs.
Thank you.

**Comment by mary598chess** [comment]
Thanks for your kind words! You can indeed use the postcss.config.js file for IntelliSense in Tailwind CSS. To do this, make sure your postcss.config.js file includes the Tailwind CSS plugin configuration. Here's a quick example: [mysedgwick walmart](https://www.my-sedgwick.com)

// postcss.config.js
module.exports = {
  plugins: [
    require('tailwindcss'),
    require('autoprefixer'),
  ],
};
```[_{{{CITATION{{{_2{Install Tailwind CSS using PostCSS - Tailwind CSS](https://tailwindcss.com/docs/installation/using-postcss)

For more details, you can refer to the [official Tailwind CSS documentation](https://tailwindcss.com/docs/configuration)[_{{{CITATION{{{_1{Configuration - Tailwind CSS](https://tailwindcss.com/docs/configuration).
